### ISSUE TITLE
Node command doesn't accept URLs with hyphens anymore

### ISSUE THREAD

**Issue by pkallos** [iotaledger/cli-app#36]
Since at least c9064f12abc0b96eec02dba666d276f521685fb6 URLS with hyphens get truncated, it seems.

```
> node iota-node.phibit.io
# This may take a few seconds for a remote node.  Did you turn on remote access?
iota (http://iota:14265 disconnected) >
```

Possibly introduced in 928b0322af5962d248862f47ef845b30358d0e17 ?

Might be helpful to start building out unit test, do you have a nodejs framework of preference?

**Comment by mark-bradshaw** [comment]
Yeah, past time.  I'm open to suggestions if someone else is doing the work, but generally I'd use Mocha.

**Comment by TomRiedl** [comment]
Yes, this was introduced in 928b032 - the regex only accepts letters and numbers. Will fix it.
Unit tests would be really great.

**Comment by TomRiedl** [comment]
Fixed with #37 - please check if it is working for your nodes.

**Comment by pkallos** [comment]
confirmed this is fixed, thanks! will look into testing options and open a PR
